src/components/AuthForm/AuthForm.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+
+import GoogleSrc from '../../assets/google-symbol.svg';
+
+import {
+    AuthFormContainer, AuthMainContainer, AuthEnterWithGoogle, GoogleIcon,
+    AuthText, InputLabel, ErrorStar, Input, ErrorMsg, AuthActionButtons, Button
+} from "./AuthForm.style";
+
+const AuthForm = ({fetchData}) => {
+    let formData = {email: null, password: null};
```

Comment:
This variable will be created on each rerender.
If you stored some data into in, and then component rerendered - formData will be {email: null, password: null}
I think ref will suit here much better)